collections/infrastructure/roles/pxe_stack/templates/Ubuntu/equipment_profile_preseed.ipxe.j2

```diff
@@ -25,13 +25,14 @@ set node-main-network-interface auto
 {% endif %}
 
 # Dynamic variables
-set images-root http://${next-server}/pxe/netboots/${eq-distribution}/${eq-distribution-version}/${eq-architecture}/
+set images-root http://${next-server}/{{ pxe_stack_images_root }}
+set images-root-iso ${images-root}/{{ pxe_stack_images_root_iso }}
 set repository-root http://${next-server}/repositories/${eq-repositories-environment}/${eq-distribution}/${eq-distribution-version}/${eq-architecture}/
-set preseed-path http://${next-server}/pxe/equipment_profiles//${eq-equipment-profile}.preseed.cfg
+set kickstart-path http://${next-server}/pxe/equipment_profiles/${eq-equipment-profile}.kickstart.cfg
 
 echo | images-root: ${images-root}
 echo | repository-root: ${repository-root}
-echo | preseed-path: ${preseed-path}
+echo | kickstart-path: ${kickstart-path}
```

Comment:
Can you restore this to preseed ?
Its ubuntu file, not kickstart

Reply:
I will do ! thanks